packages/shylu/shylu_node/basker/src/shylubasker_thread.hpp

```diff
@@ -272,7 +272,7 @@ namespace BaskerNS
     BASKER_INLINE
     void atomic_barrier_fanout(volatile Int &value, const Int l_size)
     {
-      Kokkos::atomic_increment(&(value));
+      Kokkos::atomic_inc(&(value))
```

Comment:
Note that this barrier implementation is not actually valid Kokkos code. It relies on strong forward progress of threads, and or lock-step progress assuming certain restrictions. Neither of which Kokkos semantics guarantee. I.e. this code could with a valid Kokkos backend implementation deadlock.

Reply:
I did some grepping in the code, it looks like this is long unused code (if it was ever used) so the change here is safe in the sense that this will never be called and the change keeps the code compiling with kokkos@develop